public/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Package Maintenance Team</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- load MUI -->
+    <link href="//cdn.muicss.com/mui-0.9.41/css/mui.min.css" rel="stylesheet" type="text/css" />
+    <style  type="text/css">
+      body {
+        background: #eee;
+      }
+
+      .mui-container {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 8rem;
+      }
+
+      .mui-panel {
+        padding: 2rem;
+      }
+
+      .mui-panel p {
+        font-size: 16px;
+        width: 500px;
+        margin: 1rem auto;
+      }
+    </style>
+    <script src="//cdn.muicss.com/mui-0.9.41/js/mui.min.js"></script>
+  </head>
+  <body>
+    <section class="mui-container">
+      <div class="mui-panel">
+        <h1>Package Maintenance Team</h1>
+
+        <p>Working Group for discussion on how to help ensure baseline maintenance and ability to safely use key packages in the ecosystem with current Node.js versions.
+          You can find more about this initiative in the article: Call to Action: Accelerating Node.js Growth</p>
+
+        <div class="mui-textfield">
+          <input id='package-name' type="text" placeholder="fastify">
+        </div>
+
+        <div class="mui-textfield">
+          <textarea id='badge-markdown' placeholder=""></textarea>
+        </div>
+
+        <button id='generate-badge-button' class="mui-btn mui-btn--primary mui-btn--raised">Generate Badge</button>
+      </div>
+    </section>
+
+    <script>
+      window.onload = () => {
+        document
+          .querySelector('#generate-badge-button')
+          .addEventListener('click', async () => {
+            const package = document.querySelector('#package-name').value;
+            const badge = `![package maintenance status](${window.location.href}/backing/${package})`
```

Comment:
This generate an extra slash for me:
`![package maintenance status](http://localhost:3000//backing/asd)`

And the `/backing/` part should be a `<select>` with options:
- target
- response
- backing
- response-paid
- url